Harubee-iOS/Sources/Presentation/Components/FixedExpenseListView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  FixedExpenseListView.swift
+//  Harubee-iOS
+//
+//  Created by Seo-Jooyoung on 11/2/24.
+//  Copyright © 2024 namudiEtc. All rights reserved.
+//
+
+import SwiftUI
+import DesignSystem
+
+struct FixedExpenseListView: View {
+  @State var mode: Mode = .add
+  @State private var showingSheet = false
+  
+  var body: some View {
+    VStack(spacing: 0) {
+      HStack(spacing: 0) {
+        Text("내역")
+          .font(.pretendardSemibold_16)
+        
+        Spacer()
+        
+        Button {
+          mode = .add
+          showingSheet.toggle()
+        } label: {
+          Image(systemName: "plus")
+            .frame(width: 19, height: 21)
+        }
+        .sheet(isPresented: $showingSheet) {
+          FixedExpenseManageView(mode: mode)
+            .presentationDetents([.fraction(0.8)])
+            .presentationCornerRadius(20)
+        }
```

Comment:
mode 프로퍼티는 제거해줘도 될 거 같네요.
어차피 결국 플러스 버튼 누를 때는 추가 모드, ListItemView를 눌렀을 때는 수정모드로 고정되어 있기 때문에 각 뷰에서 Sheet를 띄울 때 값을 직접 넣어주면 될 듯 합니다.
```suggestion
        
        Button {
          showingSheet = true
        } label: {
          Image(systemName: "plus")
            .frame(width: 19, height: 21)
        }
        .sheet(isPresented: $showingSheet) {
          FixedExpenseManageView(mode: .add)
            .presentationDetents([.fraction(0.8)])
            .presentationCornerRadius(20)
        }
```

Reply:
맞네요 ! 수정했습니다 !